client/src/reducers/notes.js

```diff
@@ -0,0 +1,14 @@
+import { FETCH_ALL, CREATE, DELETE } from "../constants/ActionTypes";
+
+export default (notes = [], action) => {
```

Comment:
declare separate initialState variable to hold default values

Reply:
done